### PR TITLE
ben.aubry_auth0_60be7cbc9d409a0069cd0f72-bondReference_couponRate

### PR DESCRIPTION
**Pull request by regnosys-prod-user**

### CODE REVIEW

File: rosetta-source/src/main/rosetta/product-asset-type.rosetta
Comment (ben-aubry) [inline rosetta-source/src/main/rosetta/product-asset-type.rosetta:593]: BondReference - Coupon Rate

**Background:** 
This release adds the field"couponRate" to the bondReference object under InterestRatePayout. 
The path for this field would be the following:

- InterestRatePayout > bondReference > couponRate. couponRate should be of type "FixedRateSpecification" with a cardinality of (0..1)

**Review Directions**:
- In the CDM Portal, select the Textual Browser and inspect each of the changes identified above.

**Review by iansloyan — DISMISSED** [review]
subject to release notes conforming with 
https://cdm.docs.rosetta-technology.io/source/contribution.html#content-of-release-notes

Once we have 2 approvals from ISDA  and ISLA maintainers ( @iansloyan @chrisisla ) which is sufficient to release

**Review by chrisisla — APPROVED** [review]

**Review by tomhealey-icma — COMMENTED** [review]
I think we should discuss this first. The issue I have is that this change adds specific product type attribute of an asset swap into the generic definition of an interest rate payout. I suggest either move this to a product specific type definition or change it to "UnderlyingProduct" in which case the clause boolean would move as this again is specific to a swap agreement that should possibly be in the product area or agreement area.

**Comment by iansloyan** [comment]
> I think we should discuss this first. The issue I have is that this change adds specific product type attribute of an asset swap into the generic definition of an interest rate payout. I suggest either move this to a product specific type definition or change it to "UnderlyingProduct" in which case the clause boolean would move as this again is specific to a swap agreement that should possibly be in the product area or agreement area.

Happy to discuss and may not disagree @tomhealey-icma but the BondReference type has always been there, it is that type which is being corrected to have the couponrate added. So the issue may be mutually exclusive to the change being made by @ben-aubry

**Comment by iansloyan** [comment]
@tomhealey-icma @chrisisla @iansloyan met today and agreed to proceed to merge this PR. 
A follow up issue will be raised to explore the positioning and purpose of the BondReference type in the InterestRatePayout

@PayalKhanna @nigelcobb  please proceed with merge

**Review by hugohills-regnosys — APPROVED** [review]

**Pull request merged**
